pkg/controller/pxc/vs.go

```diff
@@ -121,3 +154,13 @@ type OperatorVersion struct {
 type VersionResponse struct {
 	Versions []OperatorVersion `json:"versions"`
 }
+
+type currentVersionMeta struct {
```

Comment:
IMHO, `current` is redundant here

Reply:
but it clarify what is this version meta about

Reply:
fixed in https://github.com/percona/percona-xtradb-cluster-operator/pull/468/commits/3751f45d05c192dfffc1c2b1d4d2241b8ebd3f37